lfs.c

```diff
@@ -2476,6 +2540,7 @@ int lfs_file_opencfg(lfs_t *lfs, lfs_file_t *file,
             }
         }
 
+#ifndef LFS_READONLY
         if ((file->flags & 3) != LFS_O_RDONLY) {
```

Comment:
Related to the above, this could be `(file->flags & LFS_O_WRONLY) == LFS_O_WRONLY` for consistency

Reply:
fixed in 83fd2e8